### ISSUE TITLE
Do we have support to directly land on the dicom Study page ( not the study list )

### ISSUE THREAD

**Issue by laksmig6** [OHIF/Viewers#1476]
**Description**:

:hand: I have integrated OHIF viewer on my codebase.

Considering I have the Orthanc server set up on local machine, I have a StudyInstanceUID and the study url that I have to land on. Do I have a way to directly land on the study url not the study list.

Currently am landing directly on the study list page.

[here](https://gist.github.com/laksmig6/4928de17b395138a2705aaa7ef439f81) is the gist which has code to show how I have integrated OHIF on frontend codebase.

**Comment by dannyrb** [comment]
Yes. The application supports this, so long as you have the appropriate server redirect rules in place. The OHIF Viewer is a SPA (single page application). All network requests that don't have an exact resource match should return the `index.html`.

React's `react-router` then handles the routing on the client.

**Comment by laksmig6** [comment]
@dannyrb, Thanks for the quick response. That helped a lot and I could achieve what I was expecting.

But, am facing another problem after integrating OHIF on local. the canvas on the viewer is zooming out incrementally with the height +5px per every second. 


<img width="1440" alt="Screenshot 2020-03-04 at 8 53 43 PM" src="https://user-images.githubusercontent.com/19832110/75894760-b54cd380-5e5a-11ea-8edb-f047ce47575e.png">


[here](https://gist.github.com/laksmig6/4928de17b395138a2705aaa7ef439f81) is the gist which has code of configuration which I have used.

**Comment by dannyrb** [comment]
`https://unpkg.com/@ohif/viewer@2.0.0/dist/index.umd.js` is an old version.

Try: `https://unpkg.com/@ohif/viewer@3.7.4/dist/index.umd.js`

**Comment by laksmig6** [comment]
latest version is fixing the above issue. But looks like, this latest version has problem in hiding the study list link on the top. Am passing a boolean false to showStudyList as part of the configuration. 

[here](https://gist.github.com/laksmig6/4928de17b395138a2705aaa7ef439f81) is the gist which has code of configuration which I have used.

Some changes must be missing in the latest as I was able to hide the same with 2.0.0.
